### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by Safei-Ashraf**
Updated readme file to be more descriptive on how to use the project structure and submit issues.

### CODE REVIEW

**Review by elhaw — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -86,8 +86,65 @@ The main folder structure of the code is structured like the following:
...
+
+
+>      e.g: CompName.module.css [to be added inside the component folder]
+
+
+- Global components, hooks or logic will be stored into the "Shared" folder. e.g: shared layouts, atoms,...etc.
+
+
 
 ### The difference between the pages/ and modules/<module-name>/pages/ folders
```
Comment (AhmedAdel999) [inline README.md:127]:
```suggestion
### The difference between the pages/ and modules/\<module-name>/pages/ folders
```

**Review by AhmedAdel999 — CHANGES_REQUESTED** [review]
It doesn't render correctly as the `<` needs to be escaped.

**Review by Ali-Khamis — CHANGES_REQUESTED** [review]
change all Pickly to Pickify

**Review by Ali-Khamis — APPROVED** [review]
LGTM

**Pull request merged**